src/components/crew/crew-view/crew-view.component.js

```diff
@@ -51,12 +51,14 @@ class CrewViewPage extends Component {
 
   componentDidMount() {
     const filter = JSON.parse(this.props.match.params.filter);
-    const isCaptain =  filter["captain.license"] !== undefined;
-    let licenseNumber;
+    const isCaptain =  filter["captain.licence"] !== undefined || filter["captain.name"] !== undefined;
```

Comment:
To identify if the crew is a captain when `licence number` is absent.

Reply:
This is not working properly, and I wonder if it's because the field is captain.license (not captain.licence)?

Reply:
Sorry for this. I'll make an amend.